.circleci/config.yml

```diff
@@ -3,19 +3,13 @@ version: 2
 jobs:
   build:
     docker:
-      - image: cimg/ruby:2.7.1-node
+      - image: cimg/ruby:3.1.0-node
         environment: # environment variables for primary container
           RAILS_ENV: test
 
     steps:
       - checkout
 
-      - run:
-          name: "Update Node.js and npm"
-          command: |
-            curl -sSL "https://nodejs.org/dist/v16.9.1/node-v16.9.1-linux-x64.tar.xz" | sudo tar --strip-components=2 -xJ -C /usr/local/bin/ node-v16.9.1-linux-x64/bin/node
-            curl https://www.npmjs.com/install.sh | sudo bash
```

Comment:
The image 3.1.0 now comes with an up-to-date node version. No need for this anymore.